microsoft/testsuites/core/disk_move.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    schema,
+    search_space,
+    simple_requirement,
+)
+from lisa.features import Disk
+from lisa.features.disks import DiskStandardHDDLRS
+from lisa.node import Node
+from lisa.schema import DiskType
+from lisa.tools import Cat, Echo, Mount
+
+from .common import get_resource_disk_mount_point
+
+
+@TestSuiteMetadata(
+    area="disk storage",
+    category="functional",
+    description="""
+    Test suite exercise for ramp-up
+    """,
+)
+class DiskStorage(TestSuite):
+    @TestCaseMetadata(
+        description=""""
+        This is a demo test case for ramp up. The goal is to move a file from the home
+        folder to the data disk and assert the contents are in the disk.
+        The steps are:
+        1. Setup testcase with data disk requirement.
+        2. Write “Hello World!” string to a file.
+        3. Move the file from home folder to data disk.
+        4. Assert the file content in data disk.
+        5. Assert the file in original place does not exist.
+        """,
+        priority=4,
+        requirement=simple_requirement(
+            disk=schema.DiskOptionSettings(
+                disk_type=schema.DiskType.StandardHDDLRS,
+                data_disk_count=search_space.IntRange(min=1),
+            )
+        ),
+    )
+    def check_disk_move_operation_frmunozp(self, node: Node, log: Logger) -> None:
+        test_file = "~/test.txt"
```

Comment:
These can be a constant on the class-level

Reply:
I moved the variables to the class level.